index.js

```diff
@@ -0,0 +1,311 @@
+            break;
+        case 'Medium priority':
+            document.getElementById('Medium').checked = "true";
+            break;
+        case 'High priority':
+            document.getElementById('High').checked = "true";
+            break;
+        default:
+            document.getElementById('Low').checked = "true";
+            break;
+    }
+
+}
+
+function saveEditChanges() {
+    let li = document.querySelectorAll('.list-group-item')[index];
+    let task = {
+        title: '',
+        text: '',
+        priority: '',
+        time: ''
+    }
+    task.title = title.value;
+    task.text = text.value;
+    task.priority = selectPriority() + ' priority';
+    task.time = setTaskTime();
+    toDo.splice(index, 1, task);
+    let newTitle = taskField.children[index].getElementsByTagName("h5")[0];
+    let newText = taskField.children[index].getElementsByTagName("p")[0];
+    let newPriority = taskField.children[index].getElementsByTagName("small")[0];
+    let newTime = taskField.children[index].getElementsByTagName("small")[1];
+    newTitle.innerHTML = task.title;
+    newText.innerHTML = task.text;
+    newPriority.innerHTML = selectPriority() + ' priority';
+    newTime.innerHTML = setTaskTime();
+    colorTask(li, selectPriority() + ' priority');
+    editTaskButton.setAttribute("data-dismiss", "modal");
+    saveData();
+}
+
+function deleteTaskData() {
+    let currentTask = this.closest('.list-group-item');
+    index = Array.from(currentTask.parentNode.children).indexOf(currentTask);
+    let type = this.closest('.list-group-item').parentNode.id;
+    if (type === "currentTasks") {
+        toDo.splice(index, 1);
+        taskField.removeChild(currentTask);
+        toDoCount--
+        document.getElementById('toDo-count').innerHTML = ' (' + toDoCount + ')';
+    } else {
+        completed.splice(index, 1);
+        completeField.removeChild(currentTask);
+        completedToDoCount--
+        document.getElementById('completed-count').innerHTML = ' (' + completedToDoCount + ')';
+    }
+    saveData();
+}
+
+function sortFromNew() {
+    toDo.sort(function(a, b) {
+        return new Date(a.date) - new Date(b.date);
+    });
+    taskField.innerHTML = "";
+    toDo.forEach(function(el) {
+        return createTaskField(el, taskField);
+    })
+    saveData();
+}
+
+function sortFromOld() {
+    toDo.sort(function(a, b) {
+        return new Date(b.date) - new Date(a.date);
+    });
+    taskField.innerHTML = "";
+    toDo.forEach(function(el) {
+        return createTaskField(el, taskField);
+    })
+    saveData();
+}
+
+function changeTheme() {
+    if (switcher.checked) {
+        document.querySelector('.navbar').classList.remove('bg-light');
+        document.querySelector('.modal-content').style.backgroundColor = '#0e1621';
+        document.body.style.backgroundColor = '#0e1621';
+        document.body.style.color = 'white';
+        switcher.nextElementSibling.textContent = 'Dark Mode';

```

Comment:
all the colors, names and etc should be in according key, value object. it's applied everywhere

Reply:
I think it all must be in CSS. You should only toggle class here.

Reply:
thanks!
i fixed it!
```suggestion
document.querySelector('.modal-content').classList.add('white_theme');
```